EDDiscovery/ImageHandler.cs

```diff
@@ -819,7 +820,11 @@ private bool GetOutputSubFolder()
                     OutputFolder += "\\" + Tools.SafeFileString(CommanderID >= 0 ? EDDConfig.Instance.Commander(CommanderID).Name : "UnknownCmdr") + " at " + Tools.SafeFileString(SystemName);
                     break;
 
-                case 10: // YYYY - MM - DD CMDR name sysname
+		case 10: // CMDR name, sysname
```

Comment:
Will have their chosen setting changed without their knowledge to "CMDR name, sysname".